core/components/css-utilities/Border/Border.story.tsx

```diff
@@ -1,101 +1,95 @@
+      properties: 'border-bottom: var(--border);',
     },
     {
       className: 'border-right',
-      properties: 'border-right: var(--border) ;',
+      properties: 'border-right: var(--border);',
     },
     {
       className: 'border-left',
-      properties: 'border-left: var(--border) ;',
+      properties: 'border-left: var(--border);',
     },
     {
       className: 'border-0',
-      properties: 'border: 0 ;',
+      properties: 'border: 0;',
     },
     {
       className: 'border-top-0',
-      properties: ' border-top: 0 ;',
+      properties: 'border-top: 0;',
     },
     {
       className: 'border-bottom-0',
-      properties: 'border-bottom: 0 ;',
+      properties: 'border-bottom: 0;',
     },
     {
       className: 'border-right-0',
-      properties: 'border-right: 0 ;',
+      properties: 'border-right: 0;',
     },
     {
       className: 'border-left-0',
-      properties: 'border-left: 0 ;',
+      properties: 'border-left: 0;',
     },
   ];
 
   const radiusData = [
     {
       className: 'rounded-2-5',
-      properties: 'border-radius: 1px ;',
+      properties: 'border-radius: var(--border-radius-2-5); /* 1px */',
     },
     {
       className: 'rounded-05',
-      properties: 'border-radius: 2px ;',
+      properties: 'border-radius: var(--border-radius-05); /* 2px */',
     },
     {
       className: 'rounded-10',
-      properties: 'border-radius: 4px ;',
+      properties: 'border-radius: var(--border-radius-10); /* 4px */',
     },
     {
       className: 'rounded-15',
-      properties: 'border-radius: 6px ;',
+      properties: 'border-radius: var(--border-radius-15); /* 6px */',
     },
     {
       className: 'rounded-20',
-      properties: 'border-radius: 8px ;',
+      properties: 'border-radius: var(--border-radius-20); /* 8px */',
     },
     {
       className: 'rounded-30',
-      properties: 'border-radius: 12px ;',
+      properties: 'border-radius: var(--border-radius-30); /* 12px */',
     },
     {
       className: 'rounded-40',
-      properties: 'border-radius: 16px ;',
+      properties: 'border-radius: var(--border-radius-40); /* 16px */',
     },
     {
       className: 'rounded-full',
-      properties: 'border-radius: 9999px ;',
+      properties: 'border-radius: var(--border-radius-full); /* 9999px */',
     },
   ];
 
   return (
-    <div>
-      <Heading size="xxl">Border</Heading>
+    <div className="p-4">
+      <Heading size="xxl">Border Utilities</Heading>
       <br />
-      &nbsp;
-      <Text weight="strong">
-        Use border utilities to add or remove an element’s borders. Choose from all borders or one at a time.
-      </Text>
-      &nbsp;
-      <div
-        style={{
-          height: '250px',
-        }}
-        className="mt-5 mb-8"
-      >
+      <Paragraph appearance="default">
+        Use border utilities to add or remove an element's borders. Choose from all borders or one at a time.
+      </Paragraph>
```

Comment:
Any specific reason for updating `Text` with `Paragraph` as it does not resemble with the rest of the stories on storybook??